build.sbt

```diff
@@ -224,7 +233,7 @@ lazy val tools =
         "com.lihaoyi"    %% "scalaparse" % "0.4.2",
         "org.scalacheck" %% "scalacheck" % "1.13.4" % "test",
         compilerPlugin(
-          "org.scalamacros" % "paradise" % "2.0.1" cross CrossVersion.full),
+          "org.scalamacros" % "paradise" % "2.1.0" cross CrossVersion.full),
```

Comment:
Only 2.1.0 is available for both 2.12.3 and 2.10.6

https://repo1.maven.org/maven2/org/scalamacros/paradise_2.12.3/
https://repo1.maven.org/maven2/org/scalamacros/paradise_2.10.6/

Reply:
@xeno-by Is there a reason why macro paradise `2.1.1` is not published for `2.10.6` ?

Reply:
Oversight. I'll remedy this shortly.

Reply:
@densh any reason not to bump this, all tests are passing.

Reply:
Wow this is the definition of things falling through the cracks. I've published `2.1.1` for `2.10.7`. Would it be fine for you guys to migrate?

Reply:
`2.10.7` should work.